client/src/services/validators.test.ts

```diff
@@ -0,0 +1,131 @@
+import {
+  emailPattern,
+  epamEmailPattern,
+  englishNamePattern,
+  phonePattern,
+  urlWithIpPattern,
+  notGithubPattern,
+  githubPrUrl,
+  githubRepoUrl,
+  notUrlPattern,
+  passwordPattern,
+} from './validators';
+
+describe('email pattern', () => {
```

Comment:
Please consider to make test suites titles consistent in this file (I mean `email pattern` vs `emailPattern` stuff, IMHO, first option is preferable)

Reply:
thanks, I agree